frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -718,6 +724,30 @@ public boolean isSnapshotDisabled() {
         return snapshotDisabled;
     }
 
+    public boolean isSSLEnabled(ConnectorType connectorType) {
+        String address = prop.getProperty(TS_INFERENCE_ADDRESS, "http://127.0.0.1:8080");
```

Comment:
@dhaniram-kshirsagar  Why are the port numbers hard coded here? As these are configurable, how are things expected to work if the user picked other port numbers? Same comment for the other port numbers

Reply:
the hard coded port is default port. It is used only if user doesn't customize the ports.